commit dc31d324f038b0d11887cb917248fd3e388261b5 (refs/changes/68/368/7)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-08-29T15:16:54-07:00 (3 years, 1 month ago)
    
    trentm/play#6 add a "repository" key to package.json so npm stops complaining
    Reviewed by: undefined <undefined>
    Reviewed by: undefined <undefined>
    Approved by: undefined <undefined>
    Approved by: undefined <undefined>

diff --git a/README.md b/README.md
index 260f359..5302e3c 100644
--- a/README.md
+++ b/README.md
@@ -5,3 +5,8 @@ playin' around. nothing formal
 ## How do merges work now?
 
 Let me count the ways...
+1
+2
+3
+4
+5
diff --git a/package.json b/package.json
index 61d7d56..c75d173 100644
--- a/package.json
+++ b/package.json
@@ -4,5 +4,9 @@
   "description": "",
   "main": "lib/index.js",
   "author": "trentm@gmail.com",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/trentm/play.git"
+  },
   "license": "MIT"
 }
